common/changes/office-ui-fabric-react/zoarif-FixFocusTrapZoneFocusRegression_2017-06-01-19-21.json

```diff
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "packageName": "office-ui-fabric-react",
+      "comment": "Fixing focus trap zone previously focused element regression",
```

Comment:
fix the comment to follow the release notes format, something like

FocusTrapZone: fixing an issue where restoring focus was not working due to a regression.